Makefile

```diff
@@ -223,41 +223,51 @@ osxsnap: bundle scummvm-tools-cli-static
 #
 # AmigaOS specific
 #
-
-# Special target to create an AmigaOS snapshot installation
-amigaos4dist: all
-	mkdir -p $(AMIGAOS4PATH)
-	mkdir -p $(AMIGAOS4PATH)/graphics
-	mkdir -p $(AMIGAOS4PATH)/tools
-	mkdir -p $(AMIGAOS4PATH)/tools/media
-	cp $(srcdir)/gui/media/detaillogo.jpg $(AMIGAOS4PATH)/tools/media/
-	cp $(srcdir)/gui/media/logo.jpg $(AMIGAOS4PATH)/tools/media/
-	cp $(srcdir)/gui/media/tile.gif $(AMIGAOS4PATH)/tools/media/
-	$(STRIP) construct_mohawk$(EXEEXT) -o $(AMIGAOS4PATH)/tools/construct_mohawk$(EXEEXT)
+# Special target to create an AmigaOS snapshot installation.
+amigaosdist: all
```

Comment:
You changed the target from `amigaos4dist` to `amigaosdist` here. Is this intended? If it is so, then I'll adjust the buildbot. If it is not, I'd then revert the target name change.

Reply:
It was intended.
I'm syncronising towards scummvm main where the target is the same.

Thank you

P.s. small adaptations to the tools coming up